core/vm/interpreter.go

```diff
@@ -43,8 +43,8 @@ type MessageInterface interface {
 	GetSkipAccountChecks() bool
 }
 
-type EVMHook func(msg MessageInterface, evm *EVM) error
-
+type EVMHook func(msg MessageInterface, db StateDB) error
+type EVMReader func(tx *types.Transaction, db StateDB) (*big.Int, error) //
```

Comment:
nit delete empty `//`?

Reply:
this is to prevent `go fmt` from adding a newline before modification tag in the line below